dot/parachain/prospective-parachains/prospective-parachains.go

```diff
@@ -6,12 +6,23 @@ import (
 
 	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
 	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/lib/common"
 )
 
 var logger = log.NewFromGlobal(log.AddContext("pkg", "prospective_parachains"), log.SetLevel(log.Debug))
 
 type ProspectiveParachains struct {
 	SubsystemToOverseer chan<- any
+	View                *View
+}
+
+type View struct {
+	ActiveLeaves   map[common.Hash]bool
+	PerRelayParent map[common.Hash]*RelayParentData
+}
+
+type RelayParentData struct {
+	FragmentChains map[parachaintypes.ParaID]*fragmentChain
```

Comment:
These types are only used inside this package, so don't need to be exported, right?

```suggestion
type view struct {
	activeLeaves   map[common.Hash]bool
	perRelayParent map[common.Hash]*relayParentData
}

type relayParentData struct {
	fragmentChains map[parachaintypes.ParaID]*fragmentChain
```